app/views/common_parameters/_parameter.html.erb

```diff
@@ -22,8 +22,12 @@
                         :rows => line_count(f, :value),
                         :disabled => disabled,
                         :placeholder => _("Value")) %>
+
           <% end %>
         </td>
+        <td class="col-md-1">
```

Comment:
Yes, this is because this text area expands to the full width of the column pushing the button to the second row.